oat-sa/extension-tao-itemqti
# PNG image in preview is too big and goes beyond the page border in File Upload Interaction

**Related to task:**  https://oat-sa.atlassian.net/browse/TAO-10263
**Description:** PNG image in preview is too big and goes beyond the page border in File Upload Interaction
**How to test:** 
1. Click the Items tab
2. Create a new Item and click its "Authoring" button
3. Drag and Drop a File Upload Interaction on the canvas
4. Click the "empty space" in Interaction Properties -> MIME-type and choose PNG Image.
5. Click the "Save" button.
6. Click the "Preview" button.
7. Click "Browse..." button and select a .PNG file and observe the page
**Expected Result:**
The picture in preview doesn't go beyond the page border
